src/apps/testapps/testH3ToIj.c

```diff
@@ -38,8 +39,11 @@ void h3Distance_identity_assertions(H3Index h3) {
 
     t_assert(H3_EXPORT(h3Distance)(h3, h3) == 0, "distance to self is 0");
 
+    // Test that coordinates are as expected
+    CoordIJ ij;
+    t_assert(H3_EXPORT(h3ToIj)(h3, h3, &ij) == 0, "failed to get ij");
```

Comment:
The comment on this test doesn't make sense?

Reply:
Clarified that comment.